spec/calendar_spec.rb

```diff
@@ -49,10 +49,27 @@
       end
     end
 
-    context "when given an invalid calendar" do
+    context "when given a calendar that does not exist" do
       subject { Business::Calendar.load("invalid-calendar") }
       specify { expect { subject }.to raise_error(/No such calendar/) }
     end
+
+    context "when given a calendar that has invalid keys" do
+      subject { Business::Calendar.load("invalid-keys") }
+      specify { expect { subject }.to raise_error("Only valid keys are: holidays, working_days") }
+    end
+
+    context "when given real business data" do
+      let(:data_path) { File.join(File.dirname(__FILE__), '..', 'lib', 'business', 'data') }
```

Comment:
Should we just add this to `Business::Calendar.additional_load_paths`?

Reply:
Not following - we have it here so we can scan the directory to find all of the filenames that are available (since this directory is not exposed by the Calendar class itself). The path is present within the default load paths already.

Reply:
Sorry I wasn't clear, I meant adding this path in the `before` block!

Reply:
I think the path is already present: https://github.com/gocardless/business/blob/314a9d85870e1f20e06d597c02233bb51569368b/lib/business/calendar.rb#L11

Reply:
The path here is not used to load calendar files via the `Business` class, instead, is used to loop through the YAML files in order to execute a test for each of the files in the "data" folder, so, as Jay mentioned it is already on the load path for the class.

Reply:
👍